src/main/java/pl/edu/agh/firecell/model/material/MaterialConductionMap.java

```diff
@@ -0,0 +1,59 @@
+package pl.edu.agh.firecell.model.material;
+
+import pl.edu.agh.firecell.model.exception.ConductionCoefficientException;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class MaterialConductionMap {
+
+    private final Map<Pair, Double> coefficientMap;
+
+    public MaterialConductionMap() {
+        coefficientMap = new HashMap<>();
+    //==================================================================================================================
+        coefficientMap.put(new Pair(Material.AIR, Material.AIR),                                            /**/ 0.008);
+        coefficientMap.put(new Pair(Material.AIR, Material.WOOD),                                            /**/ 0.02);
+        coefficientMap.put(new Pair(Material.AIR, Material.CELLULAR_CONCRETE),                              /**/ 0.015);
+    //==================================================================================================================
+        coefficientMap.put(new Pair(Material.WOOD, Material.WOOD),                                            /**/ 0.2);
+        coefficientMap.put(new Pair(Material.WOOD, Material.CELLULAR_CONCRETE),                              /**/ 0.15);
+    //==================================================================================================================
+        coefficientMap.put(new Pair(Material.CELLULAR_CONCRETE, Material.CELLULAR_CONCRETE),                  /**/ 0.1);
+    //==================================================================================================================
+        try {
+            validate();
+        } catch (ConductionCoefficientException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public Double getCoefficient(Material m1, Material m2) {
```

Comment:
Assuming you create the class `UnorderedMaterialPair` which i described earlier, you should implement this method like that: create the unordered material pair from given two materials (m1 and m2) and return what inner map returns at key from the created pair.